File: campuscrib/backend/microservices/crib-manager-service/src/main/java/com/campuscrib/crib_manager_service/domain/models/Location.java
package com.campuscrib.crib_manager_service.domain.models;

import lombok.Builder;
import lombok.Getter;

import java.util.UUID;

@Builder
@Getter
public class Location {
    private UUID id;
    private String street;
    private String city;
    private String state;
    private String zipCode;
    private Double latitude;
    private Double longitude;
}

File: campuscrib/backend/microservices/crib-search-service/domain/models/location.go
package models

type Location struct {
	Latitude  float64 `bson:"latitude" json:"latitude"`
	Longitude float64 `bson:"longitude" json:"longitude"`
	Address   string  `bson:"address" json:"address"`
	City      string  `bson:"city" json:"city"`
	State     string  `bson:"state" json:"state"`
	ZipCode   string  `bson:"zip_code" json:"zip_code"`
}
